src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  --height-block: 300px;
+  display: flex;
+  flex-flow: row wrap;
+  max-width: calc(var(--height-block) * 3.99);
+}
+
+.block {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: var(--height-block);
+  min-width: var(--height-block);
```

Comment:
That's what I see on 904 px, fix it, you may want use some flex-related properties)
![image](https://user-images.githubusercontent.com/37161315/137150328-23fd5766-a65c-4aab-87f6-ac78392722df.png)